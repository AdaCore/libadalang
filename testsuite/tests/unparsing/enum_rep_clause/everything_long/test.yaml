description: |
    This test checks the formatting of an enum representation
    clause with long type name and long aggregate.
    It is expected the declaration breaks after `use` keyword
    and the aggregagte be broken, one element per new line,
    and all these new lines be indented.
 
driver: unparser
rule: aspect_clause
