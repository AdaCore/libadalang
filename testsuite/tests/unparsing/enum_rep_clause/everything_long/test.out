for VeeeeeeeeeeeeryLoooooooooooongTyyyyyyyyyyyyyyyyypeNaaaaaaaaaaaame use
  (Aaaaaaaaaaaa => 1,
   Bbbbbbbbbbb  => 5,
   Ccccccccc    => 10,
   Dddd         => 1,
   E            => 5,
   F            => 10);
